routes/api.php

```diff
@@ -60,6 +60,9 @@
             Route::any('ajax/finish', '\Djoudi\LaravelH5p\Http\Controllers\AjaxController@finish')->name("h5p.ajax.finish");
             Route::any('ajax/content-user-data', '\Djoudi\LaravelH5p\Http\Controllers\AjaxController@contentUserData')->name("h5p.ajax.content-user-data");
         });
+
+        Route::get('search', 'SearchController@search');
+        Route::get('search/advance', 'SearchController@advance');
```

Comment:
I think, "search/advance" route could be /advanced-search. @yurysakovich1010  - what do you think?

Reply:
Updated it to "search/advanced"